client-go/go.mod

```diff
@@ -0,0 +1,16 @@
+module github.com/aurae-runtime/aurae/client-go
```

Comment:
I thought the go code would be in the ae repo. It doesn't matter to me; I'm just checking that the contributors working on it (I know that includes you) are in agreement that this is the right decision at this point in time.

Side note: @krisnova @dmah42 if we make this repo have more general code, maybe we should consider moving all the rust crates into the crates directory.

Reply:
yeah i had expected this to be in a separate repo. i didn't notice this.

Reply:
What's the rational behind putting it in a different repo?

My logic for putting it in aurae-runtime/aurae was the following:

- Keep the source of truth in the same repo to avoid getting out sync
- The rust and typescript generated code are in this repo as well
- The proto files will have a `option go_package = "..."` which points to it's own repo instead of an external repo that can easily break
- If we want to move the code into another repo, we could follow kubernetes/client-go which just syncs that code from https://github.com/kubernetes/kubernetes/tree/master/staging/src/k8s.io/client-go which acts as the source of truth (and lives alongside the types it exposes)

If you all don't want it here though I can close the PR and we can figure out a more appropriate place for it.